README.md

```diff
@@ -1,24 +1,26 @@
 # factual-rules
 
-Factual rules are [YARA](https://yara.readthedocs.io/en/stable/) rules to find legitimate software on raw disk acquisition. The goal of the software is to be able to use a set of rules against collected or acquired digital forensic evidences and find installed software in a timely fashion. All the rules are generated using [factual-rules-generator](https://github.com/CIRCL/factual-rules-generator).
+Factual rules are [YARA](https://yara.readthedocs.io/en/stable/) rules to find legitimate software on raw disk acquisition.
+The goal of the software is to be able to use a set of rules against collected or acquired digital forensic evidences and find installed software in a timely fashion.
+All the rules are generated using [factual-rules-generator](https://github.com/CIRCL/factual-rules-generator).
 
 ## Source and origin of rules 
 
-YARA rules present in this repository were generated using this code ([factual-rules-generator](https://github.com/CIRCL/factual-rules-generator)). Additional
-rules can be automatically created with `factural-rules-generator` and contributed it back to this repository.
+YARA rules present in this repository were generated using scripts hosted in the [factual-rules-generator repository](https://github.com/CIRCL/factual-rules-generator).
+Additional, rules can be automatically created with `factural-rules-generator` and contributed it back to this repository.
 
 ## Rules directory format
```

Comment:
I'm not sure to understand the intent of this section. For me, it should either be simplified or removed. ;)